tests/testthat/test-mcmc.R

```diff
@@ -1,3 +1,61 @@
+# JAGS ----
+
+test_that("JAGS model compile works as expected for an example model", {
+  # The example model here is DualEndpointRW.
+  # The steps undertaken in this unit test correspond to the following commands
+  # in (command line) JAGS:
+  # jags
+  # # and then in an interactive mode
+  # model in "model.bug"
+  # data in "data.R"
```

Comment:
if the file is needed please explain in more detail which function call below needs it. also explain why the init.R is needed and where etc.